ast.c

```diff
@@ -277,10 +281,10 @@ ast_node_node_id(rb_execution_context_t *ec, VALUE self)
     return INT2FIX(nd_node_id(data->node));
 }
 
-#define NEW_CHILD(ast, node) node ? ast_new_internal(ast, node) : Qnil
+#define NEW_CHILD(vast, node) node ? ast_new_internal(vast, node) : Qnil
```

Comment:
I think a conditional expression macro should be parenthesized.

```suggestion
#define NEW_CHILD(vast, node) (node ? ast_new_internal(vast, node) : Qnil)
```